### ISSUE TITLE
GPG error: The following signatures were invalid

### ISSUE THREAD

**Issue by root7474** [dart-lang/sdk#53044]
Hi, I am trying to install Dart from the page https://dart.dev/get-dart but 'sudo apt update' give me the following:

W: GPG error: https://storage.googleapis.com/download.dartlang.org/linux/debian stable Release: The following signatures were invalid: BADSIG 4EB27DB2A3B88B8B Google Inc. (Linux Packages Signing Authority) <linux-packages-keymaster@google.com>
E: The repository 'https://storage.googleapis.com/download.dartlang.org/linux/debian stable Release' is not signed.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.

**Comment by 33b5e5** [comment]
Same on Ubuntu 22.04.02 as of today. Yesterday `apt-get update` was fine. I tried reinstalling the key:

`wget -qO- https://dl-ssl.google.com/linux/linux_signing_key.pub | sudo gpg --dearmor -o /usr/share/keyrings/dart.gpg`

However, `apt-get update` still fails:

```
W: An error occurred during the signature verification. The repository is not updated and the previous index files will be used. GPG error: https://storage.googleapis.com/download.dartlang.org/linux/debian stable Release: The following signatures were invalid: BADSIG 4EB27DB2A3B88B8B Google Inc. (Linux Packages Signing Authority) <linux-packages-keymaster@google.com>
W: Failed to fetch https://storage.googleapis.com/download.dartlang.org/linux/debian/dists/stable/Release.gpg  The following signatures were invalid: BADSIG 4EB27DB2A3B88B8B Google Inc. (Linux Packages Signing Authority) <linux-packages-keymaster@google.com>
W: Some index files failed to download. They have been ignored, or old ones used instead.
```

**Comment by whesse** [comment]
Fixed, details on #53045. It was a race condition while signing the stable and testing channels at the same time.
